src/com/spikes2212/utils/ChildNamespace.java

```diff
@@ -1,40 +1,44 @@
 package com.spikes2212.utils;
 
-import com.spikes2212.dashboard.ConstantHandler;
-
 import java.util.function.Supplier;
 
 public class ChildNamespace implements Namespace {
     private String name;
     private Namespace parent;
+    private String separator;
 
-    public ChildNamespace(String name) {
+    public ChildNamespace(String name, String separator) {
         this.name = name;
+        this.separator = separator;
     }
 
     public ChildNamespace(String name, Namespace parent) {
```

Comment:
these constructors should be unified. you have to merge the constructor with `name` and `parent` with `name` and `separator`